lib/interfaces/serve-static-options.interface.ts

```diff
@@ -89,6 +89,11 @@ export interface ServeStaticModuleOptions {
      */
     redirect?: boolean;
 
+    /**
+     * Allow multiple URLs to be served from a single root by setting this option to true when using Fastify.
+     */
+    decorateReply?: boolean;
+
```

Comment:
I guess this was not supposed to be in this PR

Reply:
Hi, this was actually the purpose of the PR. This enables multiple roots for serving static content with the Fastify Adapter. I messed up the PR by wrongfully merging master into the branch